liftoff_giftlist/src/main/resources/templates/user_details.html

```diff
@@ -1,14 +1,49 @@
 <!DOCTYPE html>
 <html lang="en" xmlns:th="http://www.thymeleaf.org/">
-<head>
-    <meta charset="UTF-8"/>
-    <title></title>
-</head>
+<head th:replace="fragments :: head"></head>
 <body>
+<nav th:replace="fragments :: nav-header"></nav>
+<div>
+<form class="mb-3" method="post">
+    <label for="firstName">First Name: </label>
+    <input type="text"
+           id="firstName"
+           name="firstName"
+           class="form-control"
+           th:value="${user.getFirstName()}"
```

Comment:
Is this approach (using th:value) preferable over using th:field? Just curious, as that's what I've been using in the forms I've written elsewhere.